modules/peers.js

```diff
@@ -813,11 +813,15 @@ Peers.prototype.onPeersReady = function() {
 				updatePeers(seriesCb) {
 					let updated = 0;
 					const peers = library.logic.peers.list();
+					library.logger.trace('Total list of peers', { count: peers.length });
 
-					library.logger.trace('Updating peers', { count: peers.length });
+					const acceptablePeers = self.acceptable(peers);
```

Comment:
We have function `pickPeers` that is calling `acceptable` and logging both total and accepted peers. Use that one.

Reply:
Why we need that at all? `acceptable` is already called during discover process (`Peers.prototype.discover`), so our own peer shouldn't reach peers manager and then shouldn't be returned from `library.logic.peers.list`. How did it get there?